core/src/server/helpers.rs

```diff
@@ -156,9 +156,10 @@ impl MethodSink {
 
 		if let Err(err) = self.send_raw(json) {
 			tracing::warn!("Error sending response {:?}", err);
+			false
```

Comment:
I think a simple two-variant enum could help understand what the `bool` returned from `send_error` specifically means